packages/fluentui/docs-components/src/knobs/useNumberKnob.ts

```diff
@@ -1,11 +1,26 @@
+import parseValue from './utils/parseRangeValue';
 import { UseKnobOptions } from './types';
 import useKnob from './useKnob';
 
-const useStringKnob = (options: UseKnobOptions<number>) =>
-  useKnob<number>({
-    initialValue: 0,
+type UseNumberKnobSpecificOptions = {
+  min?: number;
+  max?: number;
+  step?: number;
+};
+
+type UseNumberKnobOptions<T> = UseKnobOptions<T> & UseNumberKnobSpecificOptions;
+
+const useStringKnob = (options: UseNumberKnobOptions<number>) => {
+  const { initialValue = 0, min = 0, max = parseValue(initialValue), step = 1, ...rest } = options;
```

Comment:
```suggestion
  const { initialValue = 0, min = 0, max = undefined, step = 1, ...rest } = options;
```

Should it be just `undefined` to keep existing behavior?

Reply:
Done